opm/simulators/flow/BlackoilModel.hpp

```diff
@@ -130,6 +130,10 @@ template<class TypeTag>
 struct EnableDispersion<TypeTag, TTag::FlowProblem>
 { static constexpr bool value = false; };
 
+template<class TypeTag>
+struct EnableConvectiveMixing<TypeTag, TTag::FlowProblem>
+{ static constexpr bool value = true; };
```

Comment:
I didn't notice this at first.  Why is convective mixing on by default?  The prudent approach would be to have this be opt-in for users who really want that effect.

Reply:
Note that you still need to set the DRSDTCON keyword in the deck for the effect to take place.

Reply:
> Note that you still need to set the DRSDTCON keyword in the deck for the effect to take place.

Okay, but I still think that if we're adding an option to activate a new feature then that option should be turned off by default.

Reply:
I rather see this as an extension of the already existing DRSDTCON feature not a new feature and would like to avoid adding another set of simulators to the stack. The compile time option of enabling/disabling was added since this is only implemented for the Flow simulator.